pages/api/additives/[id].js

```diff
@@ -0,0 +1,21 @@
+import dbConnect from "@/db/connect";
+import Additive from "@/db/models/Additive";
+import { getToken } from "next-auth/jwt";
+
+const secret = process.env.NEXTAUTH_SECRET;
+
+export default async function handler(request, response) {
+  await dbConnect();
+  const token = await getToken({ req: request, secret });
+  const { id } = request.query;
```

Comment:
If the secret is stored in your enviroment variables, you do not need to pass it to the getToken function at this point. next-auth detects it.

Reply:
done